Civi/Token/TokenCompatSubscriber.php

```diff
@@ -268,24 +320,11 @@ public function onEvaluate(TokenValueEvent $e) {
       /** @var int $contactId */
       $contactId = $row->context['contactId'];
       if (empty($row->context['contact'])) {
-        $contact = $this->getContact($contactId, $messageTokens['contact'] ?? [], TRUE);
+        $contact = $this->getContact($contactId, $messageTokens);
```

Comment:
No longer loading 'all' here - just the 'needed' since 'all' is only needed if legacy tokens are in play & they do it